.github/workflows/tests.yaml

```diff
@@ -211,22 +211,23 @@ jobs:
                      --bridge-app ./out/linux-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
                   "
 
-            - name: Run Tests using the python parser sending commands to chip-tool
-              run: |
-                  ./scripts/run_in_build_env.sh \
-                  "./scripts/tests/run_test_suite.py \
-                     --runner chip_tool_python \
-                     --chip-tool ./out/linux-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
-                     run \
-                     --iterations 1 \
-                     --test-timeout-seconds 120 \
-                     --all-clusters-app ./out/linux-x64-all-clusters-${BUILD_VARIANT}/chip-all-clusters-app \
-                     --lock-app ./out/linux-x64-lock-${BUILD_VARIANT}/chip-lock-app \
-                     --ota-provider-app ./out/linux-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
-                     --ota-requestor-app ./out/linux-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
-                     --tv-app ./out/linux-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/linux-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                  "
+            #   https://github.com/project-chip/connectedhomeip/issues/27673
+            # - name: Run Tests using the python parser sending commands to chip-tool
```

Comment:
Why are we disabling the Linux job, if the failures we are fixing are on Darwin?

Reply:
https://github.com/project-chip/connectedhomeip/issues/27524

Reply:
OK, or https://github.com/project-chip/connectedhomeip/actions/runs/5479204916/jobs/9980729852?pr=27669 which is also a Linux fail....